docs/ref/checks.txt

```diff
@@ -239,6 +239,8 @@ Related Fields
 * **fields.E338**: The intermediary model ``<through model>`` has no field
   ``<field name>``.
 * **fields.E339**: ``<model>.<field name>`` is not a foreign key to ``<model>``.
+* **fields.E340**: The field's intermediary table name clashes with
```

Comment:
I also prefer the checks.txt version but as Tim said we should include the table name in the message. What about "The field's intermediary table ``<table name>`` clashes with ``<model>`` or ``<model>.<field name>`` table name."?